peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx

```diff
@@ -262,42 +263,53 @@ const CollaborationServiceIntegratedView: React.FC = () => {
             > Assess Code
             </button>
         </div>
+        <button
+          onClick={handleRunCode}
+          className="run-btn"
+          style={{ marginBottom: '0px' }}
+        > Run Code
+        </button>
+      </div>
 
+      <div className="code-and-chat">
         <div className="editor-container">
-            <CodeMirror
+          <CodeMirror
             ref={editorRef}
             options={{
-                mode: syntaxLang,
-                lineNumbers: true,
-                tabSize: 2,
-                indentWithTabs: true,
-                showHint: true,
-                extraKeys: {
-                    'Ctrl-Space': 'autocomplete', // Trigger autocomplete with Ctrl-Space
-                },
-                hintOptions: { completeSingle: false },
+              mode: syntaxLang,
```

Comment:
Same as above.

Reply:
that's my vs code formatting in place as per the 2103T guidelines